services/dub/dub-score.tester.js

```diff
@@ -0,0 +1,26 @@
+import Joi from 'joi'
+import { isWithinRange } from '../test-validators.js'
+import { createServiceTester } from '../tester.js'
+
+export const t = await createServiceTester()
+
+const isScoreColor = Joi.equal(
+  'red',
+  'orange',
+  'yellow',
+  'yellowgreen',
+  'green',
+  'brightgreen',
+)
+
+t.create('version (valid)')
+  .get('/vibe-d.json')
+  .expectBadge({
+    label: 'SCORE',
+    message: isWithinRange(0, 5),
```

Comment:
You can use `Joi.number()` on numeric strings. Instead of a regex, we can just use `Joi.number().min(0).max(5)` here.